Tests fail on 32-bit platforms

See the logs for [i386](https://tests.reproducible-builds.org/debian/rb-pkg/buster/i386/spyder-memory-profiler.html) and [armhf](https://tests.reproducible-builds.org/debian/rb-pkg/buster/armhf/spyder-memory-profiler.html). Both fail with the following error message:

```
============================= test session starts ==============================
platform linux -- Python 3.6.2, pytest-3.1.3, py-1.4.34, pluggy-0.4.0
PyQt5 5.7 -- Qt runtime 5.7.1 -- Qt compiled 5.7.1
rootdir: /build/spyder-memory-profiler-0.1.0, inifile:
plugins: xvfb-1.0.0, qt-2.1.2
collected 1 item

spyder_memory_profiler/widgets/tests/test_memoryprofiler.py F

=================================== FAILURES ===================================
_______________________ test_profile_and_display_results _______________________

qtbot = <pytestqt.qtbot.QtBot object at 0xae453930>
tmpdir = local('/tmp/pytest-of-pbuilder1/pytest-0/test_profile_and_display_resul0')
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0xae50bcd0>

    def test_profile_and_display_results(qtbot, tmpdir, monkeypatch):
        """Run profiler on simple script and check that results are okay."""
        os.chdir(tmpdir.strpath)
        testfilename = tmpdir.join('test_foo.py').strpath
    
        with open(testfilename, 'w') as f:
            f.write(TEST_SCRIPT)
    
        MockQMessageBox = Mock()
        monkeypatch.setattr('spyder_memory_profiler.widgets.memoryprofiler.QMessageBox',
                            MockQMessageBox)
    
        widget = MemoryProfilerWidget(None)
        qtbot.addWidget(widget)
        with qtbot.waitSignal(widget.sig_finished, timeout=10000, raising=True):
            widget.analyze(testfilename)
    
        MockQMessageBox.assert_not_called()
        dt = widget.datatree
        assert dt.topLevelItemCount() == 1  # number of functions profiled
    
        top = dt.topLevelItem(0)
        assert top.data(0, Qt.DisplayRole).startswith('foo ')
        assert top.childCount() == 6
        for i in range(6):
            assert top.child(i).data(0, Qt.DisplayRole) == i + 1  # line no
    
        # column 2 has increment (in Mib); displayed as 'xxx MiB' so need to strip
        # last 4 characters
>       assert float(top.child(2).data(2, Qt.DisplayRole)[:-4]) >= 7  # increment (MiB)
E       AssertionError: assert 3.863 >= 7
E        +  where 3.863 = float('3.863')

/build/spyder-memory-profiler-0.1.0/.pybuild/pythonX.Y_3.6/build/spyder_memory_profiler/widgets/tests/test_memoryprofiler.py:66: AssertionError
----------------------------- Captured Qt messages -----------------------------
None:None:0:
    QtWarningMsg: "QColor::setAlphaF": invalid value -0.998925
----------------------------- Captured stderr call -----------------------------
```

--- a/spyder_memory_profiler/widgets/tests/test_memoryprofiler.py
+++ b/spyder_memory_profiler/widgets/tests/test_memoryprofiler.py
@@ -9,6 +9,7 @@
 
 # Standard library imports
 import os
+import sys
 
 # Third party imports
 from pytestqt import qtbot
@@ -36,7 +37,12 @@ def foo():
 foo()"""
         
 def test_profile_and_display_results(qtbot, tmpdir, monkeypatch):
-    """Run profiler on simple script and check that results are okay."""
+    """
+    Run profiler on simple script and check that results are okay.
+
+    This is a fairly simple integration test which checks that the plugin works
+    on a basic level.
+    """
     os.chdir(tmpdir.strpath)
     testfilename = tmpdir.join('test_foo.py').strpath
 
@@ -62,12 +68,20 @@ def test_profile_and_display_results(qtbot, tmpdir, monkeypatch):
     for i in range(6):
         assert top.child(i).data(0, Qt.DisplayRole) == i + 1  # line no
 
-    # column 2 has increment (in Mib); displayed as 'xxx MiB' so need to strip
-    # last 4 characters
-    assert float(top.child(2).data(2, Qt.DisplayRole)[:-4]) >= 7  # increment (MiB)
-    assert float(top.child(2).data(2, Qt.DisplayRole)[:-4]) <= 8
-    assert float(top.child(3).data(2, Qt.DisplayRole)[:-4]) >= 150
-    assert float(top.child(3).data(2, Qt.DisplayRole)[:-4]) <= 160
-    assert float(top.child(4).data(2, Qt.DisplayRole)[:-4]) >= -160
-    assert float(top.child(4).data(2, Qt.DisplayRole)[:-4]) <= -150
+    # Column 2 has increment (in MiB); displayed as 'xxx MiB' so need to strip
+    # last 4 characters. Allow for 2% margin in measured memory consumption.
+    measured = float(top.child(2).data(2, Qt.DisplayRole)[:-4])
+    list_size_in_mib = sys.getsizeof([1] * 10 ** 6) / 2 ** 20
+    assert measured >= 0.98 * list_size_in_mib
+    assert measured <= 1.02 * list_size_in_mib
+
+    measured = float(top.child(3).data(2, Qt.DisplayRole)[:-4])
+    list_size_in_mib = sys.getsizeof([2] * 2 * 10 ** 7) / 2 ** 20
+    assert measured >= 0.98 * list_size_in_mib
+    assert measured <= 1.02 * list_size_in_mib
+
+    measured = float(top.child(4).data(2, Qt.DisplayRole)[:-4])
+    assert measured >= -1.02 * list_size_in_mib
+    assert measured <= -0.98 * list_size_in_mib
+
     assert float(top.child(5).data(2, Qt.DisplayRole)[:-4]) == 0

--- a/spyder_memory_profiler/widgets/tests/test_memoryprofiler.py
+++ b/spyder_memory_profiler/widgets/tests/test_memoryprofiler.py
@@ -7,6 +7,8 @@
 # -----------------------------------------------------------------------------
 """Tests for memoryprofiler.py."""
 
+from __future__ import division
+
 # Standard library imports
 import os
 import sys
@@ -69,19 +71,19 @@ def test_profile_and_display_results(qtbot, tmpdir, monkeypatch):
         assert top.child(i).data(0, Qt.DisplayRole) == i + 1  # line no
 
     # Column 2 has increment (in MiB); displayed as 'xxx MiB' so need to strip
-    # last 4 characters. Allow for 2% margin in measured memory consumption.
+    # last 4 characters. Allow for 5% margin in measured memory consumption.
     measured = float(top.child(2).data(2, Qt.DisplayRole)[:-4])
     list_size_in_mib = sys.getsizeof([1] * 10 ** 6) / 2 ** 20
-    assert measured >= 0.98 * list_size_in_mib
-    assert measured <= 1.02 * list_size_in_mib
+    assert measured >= 0.95 * list_size_in_mib
+    assert measured <= 1.05 * list_size_in_mib
 
     measured = float(top.child(3).data(2, Qt.DisplayRole)[:-4])
     list_size_in_mib = sys.getsizeof([2] * 2 * 10 ** 7) / 2 ** 20
-    assert measured >= 0.98 * list_size_in_mib
-    assert measured <= 1.02 * list_size_in_mib
+    assert measured >= 0.95 * list_size_in_mib
+    assert measured <= 1.05 * list_size_in_mib
 
     measured = float(top.child(4).data(2, Qt.DisplayRole)[:-4])
-    assert measured >= -1.02 * list_size_in_mib
-    assert measured <= -0.98 * list_size_in_mib
+    assert measured >= -1.05 * list_size_in_mib
+    assert measured <= -0.95 * list_size_in_mib
 
     assert float(top.child(5).data(2, Qt.DisplayRole)[:-4]) == 0

